File: LeetCode/0007_Reverse_Integer/solution.js
/**
 * @param {number} x
 * @return {number}
 */
var reverse = function(x) {
    
    var boo = false;
    if(x<0){
        boo = true;
        x *= -1;
    }
    x = Number(String(x).split('').reverse().join(''));
    if(boo) x *= -1;
    if(x>-2147483649 && x<2147483648){
    return x;}
    else return 0;
};

File: LeetCode/0007_Reverse_Integer/solution.cpp
class Solution {
public:
    int reverse(int x) {
        bool boo = false;
        long long num = x;
    if(x<0){
        boo = true;
        num *= -1;
    }
    string s=to_string(num);
    std::reverse(s.begin(),s.end());
    long long y =stoll(s);
    if(boo) y *= -1;
    if(y>-2147483649 && y<2147483648){
    return y;}
    else return 0;
    }
};